docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -111,21 +111,21 @@ const ProductsSwitcher = ({
           MUI X
         </Box>
       }
-      description="Advanced and powerful components for complex use-cases."
+      description="Advanced components for complex use cases."
```

Comment:
```suggestion
      description="Advanced components for complex use-cases."
```

Reply:
"Use case" appears to be the standard (see: [Wikipedia: Use case](https://en.wikipedia.org/wiki/Use_case)) though I have not been able to find a definitive source that explicitly spells this out.